src/widgets/splits/SplitHeader.cpp

```diff
@@ -541,59 +541,64 @@ std::unique_ptr<QMenu> SplitHeader::createMainMenu()
                             h->getDisplaySequence(HotkeyCategory::Split,
                                                   "openSubscriptionPage"));
 
-        auto *action = new QAction(this);
-        action->setText("Notify when live");
-        action->setCheckable(true);
-
-        auto notifySeq = h->getDisplaySequence(
-            HotkeyCategory::Split, "setChannelNotification", {{"toggle"}});
-        if (notifySeq.isEmpty())
         {
-            notifySeq = h->getDisplaySequence(HotkeyCategory::Split,
-                                              "setChannelNotification",
-                                              {std::vector<QString>()});
-            // this makes a full std::optional<> with an empty vector inside
-        }
-        action->setShortcut(notifySeq);
+            auto *action = new QAction(this);
+            action->setText("Notify when live");
+            action->setCheckable(true);
 
-        QObject::connect(moreMenu, &QMenu::aboutToShow, this, [action, this]() {
-            action->setChecked(getApp()->getNotifications()->isChannelNotified(
-                this->split_->getChannel()->getName(), Platform::Twitch));
-        });
-        QObject::connect(action, &QAction::triggered, this, [this]() {
-            getApp()->getNotifications()->updateChannelNotification(
-                this->split_->getChannel()->getName(), Platform::Twitch);
-        });
+            auto notifySeq = h->getDisplaySequence(
+                HotkeyCategory::Split, "setChannelNotification", {{"toggle"}});
+            if (notifySeq.isEmpty())
+            {
+                notifySeq = h->getDisplaySequence(HotkeyCategory::Split,
+                                                  "setChannelNotification",
+                                                  {std::vector<QString>()});
+                // this makes a full std::optional<> with an empty vector inside
+            }
+            action->setShortcut(notifySeq);
 
-        moreMenu->addAction(action);
-    }
+            QObject::connect(
+                moreMenu, &QMenu::aboutToShow, this, [action, this]() {
+                    action->setChecked(
+                        getApp()->getNotifications()->isChannelNotified(
+                            this->split_->getChannel()->getName(),
+                            Platform::Twitch));
+                });
+            QObject::connect(action, &QAction::triggered, this, [this]() {
+                getApp()->getNotifications()->updateChannelNotification(
+                    this->split_->getChannel()->getName(), Platform::Twitch);
+            });
 
-    if (twitchChannel)
```

Comment:
This change just means the `Mute highlight sounds` action shows up when i right click e.g. the `/mentions` tab right?

Reply:
No, this logically changes nothing, it just removes a repeated condition and replaces it with 2 empty block scopes within the same condition. The diff looks weird here, apart of the reason why it was moved to the separate PR.